### PR TITLE
Added calculator page and refactored code

### PR DESCRIPTION
**Pull request by morganbroome**
Allows for page swapping and new page creation.
NOTE: calculator is located at /calculator but patients is still current "/" (home).

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -20,32 +18,10 @@ import './styles/styles.css';
  */
  const App = () => {
  return (
-  <div className="app-container">
```
Comment (MichaelEHammond) [inline client/src/App.js:23]: We will leave this up to the front end team, but do you want routing separate from App.js?

File: client/src/App.js [hunk 2]
```diff
@@ -20,32 +18,10 @@ import './styles/styles.css';
...
-        <Routes>
-          <Route exact path="/" element={<RecordList />} />
-          <Route path="/edit/:id" element={<Edit />} />
-          <Route path="/create" element={<Create />} />
-        </Routes>
-      </div>
-    </div>
-  </div>
- );
+  <Routes>
```
Comment (MichaelEHammond) [inline client/src/App.js:21]: Why route to calculator here, but not patients? Is this the default page? If so, that works!

**Review by MichaelEHammond — APPROVED** [review]
If the routing in the App.js defines landing page, good to go!

File: client/src/App.js (hunk 2 above)
Comment (dsoman24) [reply client/src/App.js:21]: The default page later on will be the login page.

File: client/src/pages/patients.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import RecordList from "../components/recordList";
+import Edit from "../components/edit";
+import Create from "../components/create";
+import { Sidebar, Menu, MenuItem, SubMenu } from 'react-pro-sidebar';
+import { Link } from 'react-router-dom';
+ 
+const Patients = () => {
+    return (
+        <div className="app-container">
+            <Navbar />
```
Comment (dsoman24) [inline client/src/pages/patients.js:15]: Can we keep the side bar in another file and import it instead?

**Review by dsoman24 — APPROVED** [review]

**Comment by NamkhangNLe** [comment]
LG2M!

**Pull request merged**